recipes/strling/0.1.0/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "strling" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "9da4963574c44ccee529834fd594818955c367abb9083dc0fc972286d6b3f728" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quinlan-lab/STRling/releases/download/v0.1.0/strling
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
If it turns out that you need to package the upstream already compiled binary, then you should remove this dependency.